tests/testthat/test-get.R

```diff
@@ -0,0 +1,12 @@
+test_that("Has the right dimensions",{
```

Comment:
Please rename this file to test-Diet.R so that `usethis::use_test()` finds it and put the tests for all functions in Diet.R into test-Diet.R